plugins/pa11y/readme.md

```diff
@@ -0,0 +1,45 @@
+# Description 
+ Pa11y is your automated accessibility testing pal. It runs accessibility tests on your pages via the command line or Node.js, so you can automate your testing process. Pa11y runs automatically on the `test:review` and `test:staging` hooks, using the information about the app from the state.
+
+# Pa11y documentation
+ `https://github.com/pa11y/pa11y`
+
+ # Installation and usage
```

Comment:
**Issue:** We should ensure we define the correct [heading levels](https://www.markdownguide.org/basic-syntax/#headings) throughout the document. This will make it easier for the reader to scan the whole document to locate the information they need.

```suggestion
 ## Installation and usage
```